force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -316,19 +398,27 @@ public with sharing class ClassesObjectsMethods {
      * For example, personDetails() should return a list like ["Hello, my name is John Doe and I am 28 years old.", "John Doe", "28"].
      * @return A string containing the details of the person.
      */
-
     public static List<String> personDetails() {
+        
         List<String> results = new List<String>();
+        
         // Create a new instance of Person class
+        Person person = new Person(null, null);
+        
         // Name the person 'John Doe' and set the age to 28
+        person.setName('John Doe');
+        person.setAge(28);
```

Comment:
Here we can take advantage of the constructor and add the person details upon creation:
```suggestion
        Person person = new Person('John Doe', 28);
```